docs/users/odoo-openmrs.md

```diff
@@ -48,20 +50,35 @@ flowchart LR
 
     <small>_option 1_</small> &nbsp; An OpenMRS patient is always synchronised as an Odoo customer.
 
-### **2** &nbsp; OpenMRS Billable Items ⭆ Odoo Quotation
+### **2** &nbsp; OpenMRS Visit → Odoo Quotation
 
-As soon as the first billable item is ordered for a patient in OpenMRS a draft quotation is created in Odoo for the Odoo customer corresponding to that patient.
+A new Odoo quotation is linked with an OpenMRS patient's visit as soon as the first billable item is ordered during that visit.
+
+``` mermaid
+flowchart LR
+    a["OpenMRS visit"]-- 1-to-1 -->b["Odoo quotation"]
+```
+
+### **3** &nbsp; OpenMRS Billable Items ⭆ Odoo Quotation
+
+As soon as the first billable item is ordered for a patient in OpenMRS, a quotation is created in Odoo for the Odoo customer corresponding to that patient. Furthermore the quotation is linked with the OpenMRS patient's visit.
+
+All billable items ordered for a patient within the same OpenMRS visit are added to the Odoo quotation linked with this OpenMRS visit.
 
 ``` mermaid
 flowchart LR
     a["OpenMRS billable items"]-- many-to-1 -->b["Odoo quotation"]
 ```
 
-### **3** &nbsp; OpenMRS Billable Item → Odoo Quotation Item
+### **4** &nbsp; OpenMRS Billable Item → Odoo Quotation Item
+
+Each billable item ordered in OpenMRS during a patient's visit is synchronized in Odoo as a quotation item in the corresponding customer's quotation that is linked with the OpenMRS patient's visit.
 
-Each billable item ordered for an OpenMRS patient is synchronized in Odoo as an order line in the corresponding customer's quotation.
+When a drug order is discontinued for a patient in OpenMRS, the corresponding quotation item is removed from the corresponding Odoo quotation.
```

Comment:
What happens when a lab order is discontinued? This part wasn't documented in ERPNext-OpenMRS, @corneliouzbett surely the same thing happens?

Reply:
Not sure if we can discontinue a lab order. Also as per my understanding a lab order is a one time thing unlike drug order which comes under `Active Medications`
CMIIW

Reply:
> @corneliouzbett surely the same thing happens?

Yes, same thing happens.

> What happens when a lab order is discontinued?

The corresponding quotation item is removed from the quotation, and if it was the only item in the quotation, then quotation is marked as cancelled or deleted.

Reply:
@VaishSiddharth then, if you don't mind, and if this piece of documentation is missing in the Odoo-OpenMRS flows, could you add it there too as part of this PR?